### DISCUSSION TITLE
Should subpages for the package contain package name as title?

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#397] [Core Contributions] [answered]
Should all the pages of the packages in the docs folder have their package name as a title? For example, [autora-theorist-darts](https://github.com/AutoResearch/autora-theorist-darts/blob/main/docs/quickstart.md) has 'Differentiable Architecture Search' as the title in the quickstart.md file, but [autora-theotist-bsr](https://github.com/AutoResearch/autora-theorist-bsr/blob/main/docs/quickstart.md) only has it in index.md

**Comment by younesStrittmatter** [comment, marked as answer]
My opinion: Package name only in the index.md file.

**Reply by musslick** [reply]
I would also lean towards this option as it enforces fewer constraints.

**Reply by benwandrew** [reply]
agreed!
